client/my-sites/plugins/plugins-search-results-page/index.jsx

```diff
@@ -93,45 +65,29 @@ const PluginsSearchResultPage = ( {
 				}
 			);
 
-			if ( categoryName ) {
-				title = translate(
-					'Found %(total)s plugin for "%(searchTerm)s" under "%(categoryName)s"',
-					'Found %(total)s plugins for "%(searchTerm)s" under "%(categoryName)s"',
-					{
-						count: pluginsPagination.results,
-						textOnly: true,
-						args: {
-							total: pluginsPagination.results,
-							searchTerm,
-							categoryName,
-						},
-					}
-				);
-			}
+			return (
+				<>
+					<PluginsBrowserList
+						plugins={ pluginsBySearchTerm.filter( isNotBlocked ) }
+						listName={ 'plugins-browser-list__search-for_' + searchTerm.replace( /\s/g, '-' ) }
+						subtitle={
+							<>
+								{ title }
+								<ClearSearchButton />
+							</>
+						}
+						showReset={ true }
+						site={ siteSlug }
+						showPlaceholders={ isFetchingPluginsBySearchTerm }
+						currentSites={ sites }
+						variant={ PluginsBrowserListVariant.Paginated }
+						extended
+						search={ searchTerm }
+					/>
+					<InfiniteScroll nextPageMethod={ fetchNextPage } />
+				</>
+			);
 		}
```

Comment:
Eventually we are going to want to support filtering searches by category. Did we maybe go too far with the category result pages? 

We should only have one component that renders a list of search results (both keyword and category lookups are searches).

The titles (including this part) do not need to be part of that list component but we will need this text _somewhere_. Has this already been copied somewhere else? I'd rather we didn't remove it completely as we are going to want to limit searches by category eventually (we just lack the technical capability until ES integration is complete).